ts/components/session/SessionPasswordModal.tsx

```diff
@@ -182,4 +193,17 @@ export class SessionPasswordModal extends React.Component<Props, State> {
       this.props.onClose();
     }
   }
+
+  private async onKeyUp(event: any) {
+    const { onOk } = this.props;
+    const setPasswordFocussed =
+      $('#password-modal-input').is(':focus') ||
+      $('#password-modal-input-confirm').is(':focus');
```

Comment:
If you register your `this.onKeyUp()` as a callback of both of your inputs (`onKeyUp={this.onKeyUp}`) you can avoid those jquery calls